scripts/_setup_utils.js

```diff
@@ -72,17 +72,24 @@ exports.updateDotenv = function updateDotenv(add, answers) {
   );
 
   add(
-    "SECRET",
-    safeRandomString(30),
+    "SECRET_COOKIE",
+    safeRandomString(48),
     `\
 # This secret is used for signing cookies`
   );
 
   add(
-    "JWT_SECRET",
+    "SECRET_JWT_ACCESS",
+    safeRandomString(48),
+    `\
+# This secret is used for signing JWT access tokens (we don't use this by default)`
+  );
+
+  add(
+    "SECRET_JWT_REFRESH",
```

Comment:
If an app is using a JWT strategy usually it needs to encode `accessToken`s and `refreshTokens`.